projects/haproxy/project.yaml

```diff
@@ -1,9 +1,9 @@
 homepage: "https://github.com/haproxy/haproxy"
-primary_contact: "fuzzing@haproxy.org"
```

Comment:
Should fuzzing@haproxy.org be moved to auto_ccs instead? Why was it removed completely?

Reply:
Argh - actually the updates in the `project.yaml` file are not updates at all. This is the state of the `project.yaml` file already (https://github.com/google/oss-fuzz/blob/master/projects/haproxy/project.yaml), so no new changes are in there - apologies!!

Reply:
Pardon! Merged with upstream now.